server/app/mutations/stacks/common.rb

```diff
@@ -5,27 +5,16 @@ module Common
 
     include SortHelper
 
-    # @param [String] service_name
-    # @param [Symbol] key
-    # @param [Symbol] symbolic
-    # @param [String] message
-    def add_service_error(service_name, key, symbolic, message)
-      add_error("services.#{service_name}.#{key}", symbolic, message)
-    end
-
-    # @param [String] service_name
-    # @param [Hash] messages
-    # @param [String] type
+    # @param service_name [String]
+    # @param errors [Mutations::ErrorHash] Mutations::Outcome.errors
     def handle_service_outcome_errors(service_name, errors)
-      errors.each do |key, atom|
-        add_service_error(service_name, key, atom.symbolic, atom.message)
```

Comment:
I think the changes to `handle_service_outcome_errors` here would also fix the `{"env": [{"FOO": "bar"}]}` case in #2238, where the `add_service_error` currently chokes on the `Mutations::ErrorArray#symbolic` being an `[:symbol]` array.